test/e2e/signals_test.go

```diff
@@ -150,7 +150,7 @@ func (s *SignalsSuite) TestSignaledContainerSet() {
 		Workflow("@testdata/signaled-container-set-workflow.yaml").
 		When().
 		SubmitWorkflow().
-		WaitForWorkflow().
+		WaitForWorkflow(killDuration).
```

Comment:
This is to fix a flaky test: 
`    signals_test.go:153: timeout after 1m30s waiting for condition`
https://github.com/argoproj/argo-workflows/actions/runs/11907225382/job/33180716236